tests/sles4sap/trento/test_hana_unregister_node.pm

```diff
@@ -0,0 +1,82 @@
+# Copyright SUSE LLC
+# SPDX-License-Identifier: GPL-2.0-or-later
+
+# Summary: Trento unregister the secondary node
+# Maintainer: QE-SAP <qe-sap@suse.de>, Michele Pagot <michele.pagot@suse.com>
+
+use strict;
+use warnings;
+use Mojo::Base 'publiccloud::basetest';
+use base 'consoletest';
+use testapi;
+use serial_terminal 'select_serial_terminal';
+use utils 'script_retry';
+use qesapdeployment;
+use trento;
+
+sub run {
+    my ($self) = @_;
+    select_serial_terminal;
+
+    my $secondary_host = 'vmhana02';
```

Comment:
For Trento, that is Azure only, this one is fine. 

In other scenarios it does not work https://github.com/os-autoinst/os-autoinst-distri-opensuse/pull/16744

Reply:
ok master thx for the info